simtools/camera_efficiency.py

```diff
@@ -411,6 +434,11 @@ def calc_tel_efficiency(self):
     def calc_camera_efficiency(self):
         """
         Calculate the camera nominal efficiency including gaps (as defined in B-TEL-1170).
+
+        Returns
+        -------
+        cam_efficiency: float
+            Camera efficiency
```

Comment:
Is this a single value at one wavelength or an array for all wavelengths / offsets? Maybe we can be clearer here.

Reply:
It is a single value (float) as indicated. It seems to me it is a wavelength-averaged mean. can you confirm that? it is hard to interpret when the variable is called "results"

Reply:
Yes, this is correct.